src/domain/model/location.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+type Location struct {
+	Id        int      `json:"id"`
```

Comment:
By convention (most lints even show a warning) the Id fields should be renamed as ID